src/test/java/com/eta/houzezbackend/service/AgentServiceTest.java

```diff
@@ -95,12 +105,21 @@ void shouldGetActiveLinkWhenCreateSignUpLink(){
         String baseUrl = "base";
         String id = "id";
         String name = "name";
-        Integer minute = 10;
+        int minute = 10;
         String jwt = "jwt";
         when(jwtService.createJWT(id,name,minute)).thenReturn(jwt);
-        assertEquals(agentService.createSignUpLink(baseUrl,id,name,minute), baseUrl + "/verification?code=" + jwt);
+        assertEquals(agentService.createSignUpLink(baseUrl,id,name,minute), baseUrl + "/agents/decode/" + jwt);
```

Comment:
Keep the old one

Reply:
And what is this base url? Base URL for fronend or backend. We need frontend btw

Reply:
> And what is this base url? Base URL for fronend or backend？ We need frontend btw

So I need to change the base-url param from http://localhost:8080/api/v1 to frontend url in the .yml file, is that correct?

Reply:
yes